frontend/packages/employee-frontend/src/components/fee-decisions/FeeDecisionFilters.tsx

```diff
@@ -140,6 +159,32 @@ function FeeDecisionFilters() {
               .getOrElse(undefined)}
             select={selectUnit}
           />
+          <Gap size="L" />
+          <FinanceDecisionManagerFilter
+            employees={employees
+              .map((e) =>
+                e.map(({ id, firstName, lastName }) => ({
+                  id,
+                  label: [firstName, lastName].join(' ')
+                }))
+              )
+              .getOrElse([])}
+            selected={employees
+              .map(
+                (es) =>
+                  es
+                    .map(({ id, firstName, lastName }) => ({
+                      id,
+                      label: [firstName, lastName].join(' ')
+                    }))
+                    .filter(
+                      (employee) =>
+                        employee.id === searchFilters.financeDecisionManagerId
+                    )[0]
+              )
+              .getOrElse(undefined)}
```

Comment:
Similarly here, you could store a selected value so that it can be passed to the component as it is, so you don't have to iterate through the whole list every time this component is rendered.